frontend/src/store/modules/auth/getters.js

```diff
@@ -1,5 +1,9 @@
+import StorageService from '@/services/common/storageService';
+
 export default {
-    isLoggedIn: state => !!state.token,
-    getAuthenticatedUser: state => state.currentUser,
-    getToken: state => state.token,
-}
\ No newline at end of file
+    isLoggedIn: (state) => {
+        return StorageService.hasToken() || !!state.token;
```

Comment:
StorageService.hasToken() should be enough

Reply:
i have done like that but record a token to local storage does not trigger authentification. Should be another way. Investigating it